### PR TITLE
feat(ARQ-2116): Adds DbUnit configuration properties

### PR DESCRIPTION
**Pull request by lordofthejars**
Adds DbUnit configuration properties as defined at http://dbunit.sourceforge.net/properties.html

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Please also make sure that it works for in-container mode

https://github.com/arquillian/arquillian-extension-persistence/blob/1dbcc26746bee68502422bf822a2ebde34e1bec0/arquillian-ape-sql/container/int-tests/src/test/java/org/arquillian/integration/ape/dsl/ApeDslIncontainerTest.java

**Comment by lordofthejars** [comment]
@bartoszmajsak: Commit done fixing that use case.

**Comment by bartoszmajsak** [comment]
> @bartoszmajsak: Commit done fixing that use case.

This is not really needed as I'm getting notification about the commit anyway :)

File: arquillian-ape-sql/container/int-tests/src/test/resources-datasets/datasets/single-user.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-useraccount:
-      - id: 1
-        firstname: John
-        lastname: Smith
-        username: doovde
-        password: password
-        nickname: "[null]"
-        openDate: "2001-01-01"
\ No newline at end of file
+USERACCOUNT:
```
Comment (lordofthejars) [inline arquillian-ape-sql/container/int-tests/src/test/resources-datasets/datasets/single-user.yml]: Ok, I am going to run and if not I create a new file for just this case.

**Pull request merged**